### PR TITLE
feat(): Support passing the app version

### PR DESCRIPTION
**Pull request by vitorpmoreira**
## What was done:
Added a new option to the Install App console command, in order to pass in the desired app version. This is mandatory and must be a valid version for that app.
All the validations occur in the Customer Portal side.

## Todo(s):
Merge only when the Customer Portal branch `development-87270-Application_Package_Versioning` is merged to development.

### CODE REVIEW

**Review by afjsilva — COMMENTED** [review]
Is missing at the readme file the new command, explanation and respective flags

File: src/Console/InstallAppCommand.cs [hunk 1]
```diff
@@ -23,6 +23,11 @@ public InstallAppCommand(string name, string description = null) : base(name, de
                 IsRequired = true
             });
 
+            Add(new Option<string>(new[] { "--app-version" }, Resources.APP_VERSION_HELP)
```
Comment (pedroapap) [inline src/Console/InstallAppCommand.cs:26]: Would you consider adding a shortcut like -av (app version)?

Comment (vitorpmoreira) [reply src/Console/InstallAppCommand.cs:26]: We don't really have any kind of standard for doing these (besides maintaining the community standards like "-v" for "--verbose"), and I can't remember "-av" being something standard, so, yes, I can add it!

**Comment by vitorpmoreira** [comment]
> 

Documentation added!

**Comment by vitorpmoreira** [comment]
> Is missing at the readme file the new command, explanation and respective flags

Documentation added!

**Review by Rockdell — APPROVED** [review]

**Pull request merged**